cookbook/integrations/flytekit_plugins/dolt/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM python:3.8-buster
```

Comment:
couple changes are missing 
https://github.com/flyteorg/flytesnacks/blob/master/cookbook/docs/conf.py#L209
and
https://github.com/flyteorg/flytesnacks/blob/master/cookbook/dev-requirements.in#L1

if you want to run it locally